sw/supervision/pc_control_panel.ml

```diff
@@ -28,14 +28,42 @@ module Utils = Pc_common
 
 let (//) = Filename.concat
 
+(*Search recursively files in a directory*)
+let dir_contents dir =
+  let rec loop result = function
+    | f::fs when Sys.is_directory f ->
+          Sys.readdir f
```

Comment:
just in case this syntax doesn't work with older Ocaml versions: https://rosettacode.org/wiki/Walk_a_directory/Recursively#OCaml

Reply:
Done